plexpy/notification_handler.py

```diff
@@ -516,11 +563,35 @@ def build_notify_text(session=None, timeline=None, state=None):
                         'summary': metadata['summary'],
                         'tagline': metadata['tagline'],
                         'rating': metadata['rating'],
-                        'duration': duration
+                        'duration': duration,
+                        'action': state,
+                        'streams': stream_count
                         }
 
     # Default subject text
     subject_text = 'PlexPy (%s)' % server_name
+    # Default scripts args
+    script_args = ''
```

Comment:
So the "user" has to quote stuff them selfs now? I did it nice it needed for for windows dunno about other os? I liked that we quoted it for them to keep it as similar as the other notifications strings. Dunno sometime is feel I try to be too cleaver for the users.